sycl/include/sycl/ext/oneapi/bfloat16.hpp

```diff
@@ -83,8 +100,18 @@ class bfloat16 {
     return *this;
   }
 
+  // Implicit conversion from sycl::half to bfloat16
+  bfloat16(const sycl::half &a) { value = from_float(a); }
+
+  bfloat16 &operator=(const sycl::half &rhs) {
+    value = from_float(rhs);
+    return *this;
+  }
+
   // Implicit conversion from bfloat16 to float
   operator float() const { return to_float(value); }
+
+  // Implicit conversion from bfloat16 to sycl::half
   operator sycl::half() const { return to_float(value); }
 
   // Get raw bits representation of bfloat16
```

Comment:
This function `raw` is not part of the API according to the spec, so it should not be public.

Reply:
I'm going to try and remove it. I suspect some tests still use it.

Reply:
I think it would be OK to remove any tests like this.  There is no sense testing an API that we don't want.